core/common/src/PlatformUtilities.ts

```diff
@@ -0,0 +1,45 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { BentleyStatus, IModelError } from "./IModelError";
+import type { Readable, Writable } from "stream"; // Must be "import type" to avoid polyfill errors
+import type { Buffer } from "buffer"; // Must be "import type" to avoid polyfill errors
+
+const UTILITIES = Symbol.for("@itwin/core-common/PlatformUtilities");
+
+/** @internal */
+export type BackendReadable = Readable;
+
+/** @internal */
+export type BackendWritable = Writable;
+
+/** @internal */
+export type BackendBuffer = Buffer;
+
+/** @internal */
+export abstract class PlatformUtilities {
```

Comment:
Can we find something other than "PlatformUtilities" to call this class/file? Both words are confusing/ambiguous. Combined they mean nothing. There should be an explanation in this file that it is temporary and only for backwards compatibility.

Reply:
The exported types like readable/writable are temporary here.
However, the concept of a "platform utilities" will be be needed as long as we want to do things like base64 encode (which have different implementations on backend and frontend).

Reply:
If you create a garbage can like "PlatformUtilities" someone will fill it with crap. Can't you just do a dynamic import for base64?

Reply:
nothing that uses anything from "PlatformUtilities" should be in common after 4.0, right?

Reply:
Yes, we can rename to something like “BackendTypes”, remove the useful things like base64, and then it will only contain things that are slated for removal.